### PR TITLE
Update support for Laravel Framework v9

### PR DESCRIPTION
**Pull request by zaratedev**
@ErickZH 

Review this update, please!

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
         - name: Setup PHP
           uses: shivammathur/setup-php@v2
           with:
-            php-version: 7.4
+            php-version: 8.0
```
Comment (ErickZH) [inline .github/workflows/main.yml]: Change to php 8.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to `laravel-presentable` will be documented in this file
 
+## 0.2.0
+
+- Update support for Laravel Framework v9
```
Comment (ErickZH) [inline CHANGELOG.md:7]: Add new line

File: composer.json [hunk 3]
```diff
@@ -16,11 +16,11 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0",
-        "illuminate/support": "^6.0|^7.0|^8.0"
+        "php": "^8.0",
```
Comment (ErickZH) [inline composer.json]: Change to php 8.1

File: README.md [hunk 4]
```diff
@@ -11,6 +11,11 @@ This package allows the information to be presented in a different way by means
 
 You can install the package via composer:
 
+| Laravel | Version |
+| ------- | ------- |
+| 6, 7, 8 | 0.0.1   |
+| 9       | 0.2     |
```
Comment (ErickZH) [inline README.md]: Why not 0.0.2 ?

**Review by ErickZH — CHANGES_REQUESTED** [review]

**Comment by zaratedev** [comment]
@ErickZH Can you create the tag release?

**Comment by ErickZH** [comment]
@zaratedev ready!

**Pull request merged**